modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -3202,19 +3202,23 @@ private void doCheckpoint() {
 
                 if (chp.hasDelta() || destroyedPartitionsCnt > 0) {
                     if (printCheckpointStats) {
-                        if (log.isInfoEnabled())
+                        if (log.isInfoEnabled()) {
+
+
                             log.info(String.format("Checkpoint finished [cpId=%s, pages=%d, markPos=%s, " +
-                                    "walSegmentsCleared=%d, walSegmentsCovered=%s, markDuration=%dms, pagesWrite=%dms, fsync=%dms, " +
+                                    "walSegmentsCleared=%d, walSegmentsCovered=[%s - %s], markDuration=%dms, pagesWrite=%dms, fsync=%dms, " +
```

Comment:
Will we print `[0 - 0]` here if no wal segmetns covered?

Reply:
I improved implementation so it will handle corner cases as follows:

- if there is no segments covered, empty square brackets will be printed out: []

- if only one segment covered, it won't be duplicated: [6]

- range will be printed if more than one segment if covered: [0 - 1]